tests/ecosystem/upgrade/test_upgrade.py

```diff
@@ -21,11 +23,83 @@
 log = logging.getLogger(__name__)
 
 
+def get_upgrade_image_info(old_csv_images, new_csv_images):
+    """
+    Log the info about the images which are going to be upgraded.
+
+    Args:
+        old_csv_images (dict): dictionary with the old images from pre upgrade CSV
+        new_csv_images (dict): dictionary with the post upgrade images
+
+    Returns:
+        tuple: with three sets which contain those types of images
+            old_images_for_upgrade - images which should be upgraded
+            new_images_to_upgrade - new images to which upgrade
+            unchanged_images - unchanged images without upgrade
+
+    """
+    old_csv_images = set(old_csv_images.values())
+    new_csv_images = set(new_csv_images.values())
+    old_images_for_upgrade = old_csv_images - new_csv_images
+    log.info(
+        f"Old images which are going to be upgraded: "
+        f"{sorted(old_images_for_upgrade)}"
+    )
+    new_images_to_upgrade = new_csv_images - old_csv_images
+    log.info(
+        f"New images for upgrade: "
+        f"{sorted(new_images_to_upgrade)}"
+    )
+    unchanged_images = old_csv_images.intersection(new_csv_images)
+    log.info(
+        f"Unchanged images after upgrade: "
+        f"{sorted(unchanged_images)}"
+    )
+    return (
+        old_images_for_upgrade,
+        new_images_to_upgrade,
+        unchanged_images,
+    )
+
+
+def verify_image_versions(old_images):
+    """
+    Verify if all the images of OCS objects got upgraded
+
+    Args:
+        old_images (dict): dictionary with old images
+
+    """
+    verify_pods_upgraded(old_images, selector=constants.OCS_OPERATOR_LABEL)
```

Comment:
old_images is of dict datatype whereas verify_pods_upgraded needs set datatype.
Looks like mismatch of datatype

Reply:
Yep, here suppose to be also set, will update the docstring. Thanks